docs/install/install-cave.md

```diff
@@ -60,25 +60,39 @@ To run CAVE either:
 - Use the terminal and type the command `cave`
 - Find the application in the Linux Desktop menu: Applications > Internet > AWIPS CAVE
 
-Additionally users can choose to run a [virtual machine (VM)](#method-1-linux-virtual-machine) on Linux.
+Additionally users can choose to run a [virtual machine (VM)](#method-2-linux-virtual-machine) on Linux.
 
 ---
 
 ## Windows <i class="fa fa-windows"></i> 
 
 **Latest Version: 18.2.1-6**
 
-For Windows, Unidata offers two installation options: a [**Linux Virtual Machine**](#method-1-linux-virtual-machine), or a [**Direct Windows Installation**](#method-2-direct-windows-install).
+For Windows, Unidata offers two installation options: a [**Direct Windows Installation**](#method-1-direct-windows-install), or a [**Linux Virtual Machine**](#method-2-linux-virtual-machine).
+
+!!! note "As of 4/24/2023, the direct install method has been completely simplified! No admin priviledges or extra software needed."
 
-Currently, the [virtual machine (VM)](#method-1-linux-virtual-machine) is the recommended form of install for those who do not have administrative priviledges on the machine, or beginners who want a simpler installation process.  
 !!! warning "At the moment, the VM option may not render all products in CAVE (ex. RGB composites of satellite imagery)"
 
-The [direct installation method](#method-2-direct-windows-install) is recommended for those who have administrative priviledges and a little bit of experience installing more software.
+### Method 1: Direct Windows Install
+
+This method has been simplified to include python  and java packaged with CAVE so no other software installation is necessary. CAVE no longer needs manual changes to any environment variables in order to run.
```

Comment:
"CAVE no longer needs manual changes to any environment variables in order to run."

This sort of implies it still modifies system/user environment variables.

Maybe, "CAVE no longer needs environment variable changes in order to run."

Reply:
I mean on execution it does modify the user environment variables, just not permanently, but I can remove it. I was just trying to tell the users how it had changed and now works (which was why I originally had the bit about the .bat file)

Reply:
It modifies the environment variables of the executable it's running in.  It doesn't modify the User's variables.